packages/account/build/webpack.config.js

```diff
@@ -14,14 +14,14 @@ module.exports = function (env) {
             'address-details': 'Components/address-details',
             'address-details-config': 'Configs/address-details-config',
             'api-token': 'Components/api-token',
+            'currency-radio-button': 'Components/currency-selector/radio-button.jsx',
+            'currency-radio-button-group': 'Components/currency-selector/radio-button-group.jsx',
             'currency-selector': 'Components/currency-selector',
```

Comment:
lines just sorted alphabetically,
added file-uploader, current uploader-container renamed to poa-file-uploader-container